juniper_official/Solutions/EVPN-VXLAN/spine-evpn-vxlan-kpis.playbook

```diff
@@ -0,0 +1,74 @@
+/*
+ * Playbook contains multiple rules which monitors Data Center Spine
+ * nodes and notifies when anomalies are found.
+ *
+ * 1) Rule evpn-advertised-prefix.rule, detects the advertised route count
+ *    threshold breaches and notify anomalies.
+ * 2) Rule evpn-received-prefix.rule, detects the received route count
+ *    threshold breaches and notify anomalies.
+ * 3) Rule evpn-route-count.rule, detects the route count threshold
+ *    breaches and notify anomalies.
+ * 4) Rule evpn-mac-count.rule, detects the evpn mac count threshold
+ *    breaches and notify anomalies.
+ * 5) Rule bfd-session-state.rule, detects BFD session state change and
+ *    notifies anomalies when error count increases.
+ * 6) Rule vtep-interface-count.rule, detects the vtep interface  count
+ *    threshold breaches and notify anomalies.
+ * 7) Rule vtep-interface-status.rule, detects the vtep interface status
+ *    notify anomalies.
+ * 8) Rule vtep-interface-traffic.rule,monitors interface traffic and notifies
+      anomalies when traffic exceeds threshold.
+ * 9) Rule vtep-interface-flap.rule, detects interface flaps and notifies
+ *    anomalies when link flap count increases.
+ * 10) Rule vtep-interface-error.rule, detects interface errors and notifies
+ *     anomalies when error count increases.
+ * 11) Rule vtep-tunnel-statistics.rule, detects logical interface statisticsr
+ *     and notifies anomalies when error count increases.
+ * 12) Rule irb-interface-status.rule, detects the irb interface status
+ *     notify anomalies.
```

Comment:
s/Rule irb-interface-status.rule, detects the irb interface status notify anomalies/Rule irb-interface-status.rule, monitors irb interface and notify anomaly when irb state is down/g

Reply:
updated the rule